### PR TITLE
feat: add link to Selv demo

### PR DESCRIPTION
**Pull request by daniel-mader**
# Description of change
Adds a link to the Selv demo and adjusts how the links are presented.

## Links to any relevant issues
n/a

## How the change has been tested
n/a

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by daniel-mader** [comment]
Screenshot 2024-06-06 at 23 28 07
Screenshot 2024-06-06 at 23 28 24

File: unime/src/i18n/de-DE/index.ts [hunk 1]
```diff
@@ -180,10 +180,7 @@ const de = {
       GREETING_3: 'Hallo',
       GREETING_4: 'Hi',
     },
-    DEMO: {
-      TEXT_1: 'Besuche',
-      TEXT_2: 'auf einem Desktop-Computer um loszulegen.',
-    },
+    DEMO: 'Besuche eine der folgenden Webseiten auf einem Desktop-Computer um loszulegen.',
```
Comment (maiertech) [inline unime/src/i18n/de-DE/index.ts]: Comma after Desktop-Computer.

File: unime/src/routes/(app)/me/+page.svelte
Comment (SoulaAndrikopoulos) [inline unime/src/routes/(app)/me/+page.svelte:1]: This warning appears `{@html}` can lead to XSS attack' in some different files as well.

**Review by SoulaAndrikopoulos — DISMISSED** [review]

**Comment by SoulaAndrikopoulos** [comment]
Styling looks good in both dark mode and light mode. We just have to make sure to test this with demo users, and see if users will understand that the links are not clickable. 👍

File: unime/src/routes/(app)/me/+page.svelte [hunk 2]
```diff
@@ -34,6 +39,12 @@
     }
   }
 
+  // Explicitly listen for dark mode changes, so that icons can react to it
```
Comment (maiertech) [inline unime/src/routes/(app)/me/+page.svelte]: The icon's own styles should adapt their styling based soley on the presense or absense of the global `dark` class, not on a flag. The `dark` class on the html element is the flag.

File: unime/src/routes/(app)/me/+page.svelte [hunk 3]
```diff
@@ -166,10 +177,29 @@
       <!-- Skipped onboarding journey -->
       <div class="flex grow flex-col items-center justify-center">
         <IconMessage icon={Ghost} title={$LL.ME.EMPTY_CREDENTIALS.TITLE()} />
-        <p class="w-[280px] pt-[15px] text-center text-[13px]/[24px] font-normal text-slate-500 dark:text-slate-300">
-          {$LL.ME.DEMO.TEXT_1()} <span class="font-semibold text-primary">https://demo.ngdil.com</span>
-          {$LL.ME.DEMO.TEXT_2()}
-        </p>
+        <div class="w-[280px] pt-[15px] text-center text-[13px]/[24px] font-normal text-slate-500 dark:text-slate-300">
+          {$LL.ME.DEMO()}
+          <div class="flex flex-col gap-3 pt-[15px]">
```
Comment (maiertech) [inline unime/src/routes/(app)/me/+page.svelte:173]: There should be only one SVG for each icon. The SVG can adapt it's styling based on the global `dark` class.

File: unime/src/lib/credentials/CredentialList.svelte [hunk 4]
```diff
@@ -46,9 +46,12 @@
   <!-- Only show "No credentials" when there's also no favorites -->
```
Comment (maiertech) [inline unime/src/lib/credentials/CredentialList.svelte:46]: In the `if` you evaluate the **filtered** credentials. Here you evaluate **all** credentials. When there are no credentials at all, the the `/me` route does not render `CredentialList`. This part does not seem to ever be rendered.

**Review by maiertech — APPROVED** [review]

**Review by maiertech — CHANGES_REQUESTED** [review]
This seems more complicated than it should be.

File: unime/src/routes/(app)/me/+page.svelte
Comment (daniel-mader) [reply unime/src/routes/(app)/me/+page.svelte:1]: Yeah, an attack is highly unlikely in our use case (probably impossible at all) - it's a warning and we are aware of it, but we decided not to silence it just yet. 👍

File: unime/src/routes/(app)/me/+page.svelte (hunk 3 above)
Comment (daniel-mader) [reply unime/src/routes/(app)/me/+page.svelte:173]: Refactored 👍 Thanks for pointing it out!

**Review by SoulaAndrikopoulos — APPROVED** [review]

File: unime/src/lib/credentials/CredentialList.svelte (hunk 4 above)
Comment (maiertech) [reply unime/src/lib/credentials/CredentialList.svelte:46]: Will be tackled in https://github.com/impierce/identity-wallet/issues/253.

**Review by maiertech — APPROVED** [review]

**Pull request merged**